### PR TITLE
Fixes Issue 60 : updated the count to 2 in js

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
Solves https://github.com/devict/hacktoberfest/issues/60

### CODE REVIEW

File: public/js/profile.js [hunk 1]
```diff
@@ -87,11 +87,11 @@ function checkPRs() {
       } else if (validCount === 0) {
         // Some PRs but none count
         message = 'You have ' + data.length + ' Pull Request(s) but none of them are against approved repos.';
-      } else if (validCount < 4) {
+      } else if (validCount < 2) {
         // Some PRs that count but not quite 4
```
Comment (southekal) [inline public/js/profile.js]: thanks! can you please update the number in the comment also please.

**Comment by M-ZubairAhmed** [comment]
Added the change @southekal

**Comment by M-ZubairAhmed** [comment]
Why is this closed?  And not merged

**Comment by southekal** [comment]
my bad @M-ZubairAhmed 
for some reason I thought it was merged in. thanks

**Pull request merged**